examples/dbclient/common/src/main/java/io/helidon/examples/dbclient/common/AbstractPokemonService.java

```diff
@@ -124,74 +120,116 @@ private void insertPokemonSimple(ServerRequest request, ServerResponse response)
      * @param request  server request
      * @param response server response
      */
-    private void getPokemon(ServerRequest request, ServerResponse response) {
+    private void getPokemonByName(ServerRequest request, ServerResponse response) {
         String pokemonName = request.path().param("name");
-
-        dbClient.execute(exec -> exec.namedGet("select-one", pokemonName))
+        dbClient.execute(exec -> exec.namedGet("select-pokemon-by-name", pokemonName))
                 .onEmpty(() -> sendNotFound(response, "Pokemon " + pokemonName + " not found"))
                 .onValue(row -> sendRow(row, response))
                 .exceptionally(throwable -> sendError(throwable, response));
     }
 
     /**
-     * Return JsonArray with all stored pokemons or pokemons with matching attributes.
+     * Insert new pokemon with specified name.
      *
      * @param request  the server request
      * @param response the server response
      */
-    private void listPokemons(ServerRequest request, ServerResponse response) {
-        dbClient.execute(exec -> exec.namedQuery("select-all"))
-                .thenAccept(response::send)
+    private void insertPokemon(ServerRequest request, ServerResponse response, Pokemon pokemon) {
+        dbClient.execute(exec -> exec
+                .createNamedInsert("insert-pokemon")
+                .indexedParam(pokemon)
+                .execute())
+                .thenAccept(count -> response.send("Inserted: " + count + " values"))
                 .exceptionally(throwable -> sendError(throwable, response));
     }
 
+    /**
+     * Insert new pokemon type to pokemon with specified id.
+     *
+     * @param request  the server request
+     * @param response the server response
+     */
+    private void insertPokemonType(ServerRequest request, ServerResponse response) {
+        try {
+            int pokemonId = Integer.parseInt(request.path().param("pokemonId"));
+            int typeId = Integer.parseInt(request.path().param("typeId"));
+            dbClient.execute(exec -> exec
+                    .namedInsert("insert-poketype", pokemonId, typeId))
+                    .thenAccept(count -> response.send("Inserted: " + count + " values"))
+                    .exceptionally(throwable -> sendError(throwable, response));
+        } catch (NumberFormatException ex) {
+            sendError(ex, response);
+        }
+    }
+
     /**
      * Update a pokemon.
      * Uses a transaction.
      *
      * @param request  the server request
      * @param response the server response
      */
-    private void updatePokemonType(ServerRequest request, ServerResponse response) {
+    private void updatePokemonName(ServerRequest request, ServerResponse response) {
+        int id = Integer.parseInt(request.path().param("id"));
```

Comment:
you catch the `NumberFormatException` above, not here. Is there a reason not to be consistent?